salt/top.sls

```diff
@@ -33,7 +33,3 @@ base:
     - kube-apiserver
     - kube-controller-manager
     - kube-scheduler
-    - addons
-    - addons.dns
-    - addons.tiller
-    - dex
```

Comment:
We shoudn't be removing these from the highstate as the highstate should apply all states related to the node, the orchestration is purely about giving some ordering to allow for first time setup etc.

Reply:
Oh. I see why you're doing this. Right - these don't actually "belong" to a master, they belong to the cluster as a whole. Humm. Fair enough :)

Reply:
I think we had this discussion before. These things are not intrinsic to the Masters: they are cluster services. Making them something tighted to a master leads to this problem where we try to install them multiple times.